common/macresman.cpp

```diff
@@ -403,6 +411,25 @@ bool MacResManager::loadFromMacBinary(SeekableReadStream *stream) {
 			_resForkOffset = MBI_INFOHDR + dataSizePad;
 			_resForkSize = rsrcSize;
 		}
+
+		if (_resForkSize == 0) {
+			debug(7, "resource-less macbinary");
+
+			_mode = kResForkMacBinary;
```

Comment:
I do not understand the meaning of this flag. The condition above is meant when there is NO resource fork, but this flag says "mac binary with resource fork". Or am I misreading it? I think we need to clarify or give the flag a better name.

Reply:
Again Mac binary just like real Mac files always has resource fork. Possibly zero-sized. We really have resource-less files only after bad copy that killed them.